WcaOnRails/app/webpacker/components/Panel/Wrt/EditPerson.jsx

```diff
@@ -0,0 +1,183 @@
+import React, { useState } from 'react';
+import DatePicker from 'react-datepicker';
+import {
+  Button, Form, Icon, Message,
+} from 'semantic-ui-react';
+import _ from 'lodash';
+import { adminUpdatePersonUrl, adminCheckRecordsUrl } from '../../../lib/requests/routes.js.erb';
+import useSaveAction from '../../../lib/hooks/useSaveAction';
+import WcaSearch from '../../SearchWidget/WcaSearch';
+import Loading from '../../Requests/Loading';
+import I18n from '../../../lib/i18n';
+import { genders } from '../../../lib/wca-data.js.erb';
+import 'react-datepicker/dist/react-datepicker.css';
+
+const description = `
+  <p>
+  Choose 'Fix' if you want to replace a person's information in the database.
+  It will modify the Persons table accordingly and the Results table if the person's name is different.
+  This should be used to fix mistakes in the database.
+  </p>
+
+  <p>
+  Choose 'Update' if the person's name or country has been changed.
+  It will add a new entry in the Persons table and make it the current information for that person (subId=1)
+  but it will not modify the Results table so previous results keep the old name.
+  </p>
+`;
```

Comment:
Why are you encoding this to be a string and then dangerously setting inner HTML below?
This should be directly in the HTML source returned by the React component (unless I'm overseeing a *very* good reason to have it be separate)

Reply:
Okay, sorry that @coder13 and @kr-matthews has already mentioned this in https://github.com/thewca/worldcubeassociation.org/pull/8406#discussion_r1398233362, but I forgot to address it further.

Reply:
I've also changed the text of description to avoid duplication of explanation in two places.